internal/test/framework/windowsvm.go

```diff
@@ -146,6 +149,63 @@ func (w *windowsVM) CopyFile(filePath, remoteDir string) error {
 	return nil
 }
 
+// RetrieveFiles retrieves list of files from remote directory to the local directory.
+// As of now, I think there is no need to have a recursive copy as the log directory should contain only log files
+//but no child directories. I am fine change the implementation if the use-case arises.
```

Comment:
Remove `I am fine` and make it a statement. `The implementation can be changed if the use-case arises`.